shared_model/packages/javascript/scripts/clean.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+echo "Cleaning all generated directories and files..."
```

Comment:
Without confirmation looks redundant

Reply:
No error during script invocation is a confirmation.